utils.py

```diff
@@ -52,6 +52,17 @@ def send_email(logger, subject, msg, recipient=None, attachment=None):
     except Exception as ex: # pylint: disable=broad-except
         logger.error(f"email '{subject}': {ex}")
 
+def send_sms(logger, subject, msg, recipient=None, attachment=None):
```

Comment:
fixes the admin from being able to send out to phone no.. mobile messages only dispays the word "Zap" on the message with no link before.... this fixes that issue.